addons/skin.confluence/language/resource.language.en_gb/strings.po

```diff
@@ -118,6 +118,10 @@ msgctxt "#31040"
 msgid "Now playing"
 msgstr ""
 
+msgctxt "#31041"
+msgid "Timeshifting: "
```

Comment:
was thinking about that as well. In any way we would need a function to glue strings depending on if it's a RTL language or not.